docs/user-guides/cb-and-fallback.md

```diff
@@ -11,9 +11,11 @@ it also monitor each service call to make service [observable](https://docs.go-c
 Circuit breaker scope is controlled by 
 
 **cse.circuitBreaker.scope**
-> *(optional, string)* service or api, 
+> *(optional, string)* service、instance or api, 
 default is api, go chassis create a dedicated circuit for every api, invocation will be isolated based on api. 
 if set to service, all of APIs of each service share one circuit, it will isolate the service.
+if set to instance, all of APIs of each instance share one circuit, it will isolate the instance.
+if set to instance,api, each api of each instance isolate one circuit.
```

Comment:
if set to instance,  each instance will get a dedicated circuit, it will isolate only one instance.

Reply:
你意思是把
```
if set to instance, all of APIs of each instance share one circuit, it will isolate the instance.
```
替换为这句？
```
if set to instance, each instance will get a dedicated circuit, it will isolate only one instance.
```

Reply:
还是只用这一句：
```
if set to instance, each instance will get a dedicated circuit, it will isolate only one instance.
```

Reply:
你的两句替换成这两句

if set to instance, each instance will get a dedicated circuit, it will isolate only one instance.

if set to api , each api will get a dedicated circuit, it will isolate only service api.